### PR TITLE
Email confirmation fixed issues

### PR DESCRIPTION
**Pull request by newcitywave**

### CODE REVIEW

File: src/pages/ConfirmationPage.js [hunk 1]
```diff
@@ -70,20 +74,33 @@ class ConfirmationPage extends React.Component {
...
+  //     chr
+  //   const keys = email + paperFileNumber
+  //   if (keys.length === 0) return hash
+  //   for (i = 0; i < keys.length; i++) {
+  //     chr = keys.charCodeAt(i)
+  //     hash = (hash << 5) - hash + chr
+  //     hash |= 0
+  //   }
+  //   return hash
+  // }
```
Comment (var-kyle) [inline src/pages/ConfirmationPage.js]: There shouldn't be a need to keep commented out code.

File: src/pages/ReviewPage.js [hunk 2]
```diff
@@ -25,47 +25,37 @@ const spacingButton = css`
...
+  //     chr
+  //   const keys = email + paperFileNumber
+  //   if (keys.length === 0) return hash
+  //   for (i = 0; i < keys.length; i++) {
+  //     chr = keys.charCodeAt(i)
+  //     hash = (hash << 5) - hash + chr
+  //     hash |= 0
+  //   }
+  //   return hash
+  // }
```
Comment (var-kyle) [inline src/pages/ReviewPage.js]: Same as above, keeping this should be unnecessary.

File: src/pages/ConfirmationPage.js (hunk 1 above)
Comment (will0684) [reply src/pages/ConfirmationPage.js]: Looks like this was removed

File: src/pages/ReviewPage.js (hunk 2 above)
Comment (will0684) [reply src/pages/ReviewPage.js]: Comments have been removed

**Pull request closed without merging**